client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -47,19 +47,32 @@ import Banner from 'components/banner';
 import { isEnabled } from 'config';
 import wpcomFeaturesAsPlugins from './wpcom-features-as-plugins';
 
-const PluginsBrowser = createReactClass( {
-	displayName: 'PluginsBrowser',
-	_SHORT_LIST_LENGTH: 6,
-	visibleCategories: [ 'new', 'popular', 'featured' ],
-	mixins: [ URLSearch ],
+/**
+ * Module variables
+ */
+const SHORT_LIST_LENGTH = 6;
+const visibleCategories = [ 'new', 'popular', 'featured' ];
+
+export class PluginsBrowser extends Component {
+	static displayName = 'PluginsBrowser';
+
+	constructor( props ) {
+		super( props );
+
+		const pluginsList = this.getPluginsLists( this.props.search );
+
+		this.state = {
+			...pluginsList,
+		};
+	}
```

Comment:
Can we ditch the constructor and use the property initializer shorthand for `state` here?

```jsx
export class PluginsBrowser extends Component {
	static displayName = 'PluginsBrowser';

	state = ...this.getPluginsLists( this.props.search );

	/*...*/
}
```

Reply:
Sorted, cheers @ockham :)